build/install-gentoo

```diff
@@ -16,27 +16,54 @@
 
 # Check best compilation flags for GCC
 export CC="gcc"
-export CFLAGS="-march=native -mtune=native -O2 -fno-delete-null-pointer-checks -finline-functions -fomit-frame-pointer -fno-builtin-memcmp"
+export CFLAGS="-march=native -mtune=native -O2 -fomit-frame-pointer"
+export CPPFLAGS="-DPHALCON_RELEASE"
+
+#PHP_CONFIG=`php-config --version`
+
+PHP_FULL_VERSION=`php-config --version`
+if [ $? != 0 ]; then
+	echo "php-config is not installed"
+	exit 1
+fi
+
+if [ "${PHP_FULL_VERSION:0:3}" == "5.3" ]; then
+	echo "php 5.3 is no longer supported"
+	exit 1
```

Comment:
@Jurigag This is what the original code in build/install for building 3.0.x version does, so I have just ported it over. I have not tested compiling it or anything on PHP 5.3 and 5.4